pyfakefs/fake_pathlib.py

```diff
@@ -650,17 +650,25 @@ def read_bytes(self):
             OSError: if the target object is a directory, the path is
                 invalid or permission is denied.
         """
-        with FakeFileOpen(self.filesystem)(
-            self._path(), mode="rb"
-        ) as f:  # pytype: disable=attribute-error
+        with fake_open(
+            self.filesystem,
+            self.skip_names,
+            self._path(),
+            mode="rb",
+        ) as f:
             return f.read()
```

Comment:
Ah, thanks - I was thinking about this and completely forgot that!

Reply:
I'm glad I could contribute!
Can you kindly take care of the release notes? I'm not sure about the release date.

Reply:
Thanks a lot!

Ah, there is no entry after the release yet. Just put `## Unreleased` on top instead of a release description, no release date is needed - that will only be added at release time. And `pytype` is still failing... it can be a bit finicky.

Reply:
Let me know if I shall handle the `pytype` errors myself.

Reply:
I'm adding `__enter__` and `__exit__` to some classes like `FakeDirWrapper`, I'm following the patterns in `FakePipeWrapper`.
If it needs to be handled differently, I'd be thankful if you take care of it yourself.

Reply:
Does `read` have a meaning for `FakeDirWrapper`?

Reply:
I think this is the correct way to handle this, thanks!

Reply:
I tried to add `read` and `write` methods in a meaningful way, as far as I could understand the semantics. The tests are now passing and I added the release entry, please review my changes when you got the time.